packages/nouns-contracts/contracts/client-incentives/INounsClientTokenTypes.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: GPL-3.0
+
+/*********************************
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░█████████░░█████████░░░ *
+ * ░░░░░░██░░░████░░██░░░████░░░ *
+ * ░░██████░░░████████░░░████░░░ *
+ * ░░██░░██░░░████░░██░░░████░░░ *
+ * ░░██░░██░░░████░░██░░░████░░░ *
+ * ░░░░░░█████████░░█████████░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ *********************************/
+
+pragma solidity ^0.8.19;
+
+interface INounsClientTokenTypes {
+    struct ClientMetadata {
+        string name;
+        string description;
+    }
+
+    struct TokenState {
```

Comment:
maybe rename to `Storage` to indicate this is the entire storage of the token

Reply:
done